From 2c274d28e8bbce620c76b597845aac7b2d479ffc Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 16 Oct 2020 18:39:09 +0300
Subject: [PATCH 2/3] arm64: boot: dts: renesas: ulcb-kf: Enable extra sound
 cards

This enables pcm3168a, si468x-pcm radio, and wl18xx-pcm BT sound cards.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/ulcb-kf.dtsi | 27 ++++++++-------------------
 1 file changed, 8 insertions(+), 19 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
index d5ada3a..0132644 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
@@ -143,14 +143,14 @@
 		compatible = "btwilink";
 	};
 
-#if 0
 	/delete-node/sound;
-	sound_card: sound@1 {
+
+	sound_card: sound@0 {
 		compatible = "audio-graph-card";
 		label = "ak4613";
 
-		dais = <&rsnd_port2	/* ak4613 */
-			&rsnd_port3	/* HDMI0  */
+		dais = <&rsnd_port0	/* ak4613 */
+			&rsnd_port1	/* HDMI0  */
 			>;
 	};
 
@@ -158,12 +158,11 @@
 		compatible = "audio-graph-card";
 		label = "pcm3168a";
 
-		dais = <&rsnd_port0	/* playback */
-			&rsnd_port1	/* capture  */
+		dais = <&rsnd_port2	/* playback */
+			&rsnd_port3	/* capture  */
 			>;
 	};
-#endif
-#if 0
+
 	sound_radio: sound@2 {
 		compatible = "audio-graph-card";
 		label = "radio";
@@ -177,7 +176,7 @@
 
 		dais = <&rsnd_port5>;
 	};
-#endif
+
 	lvds: lvds {
 		port {
 			lvds_in: endpoint {
@@ -1115,16 +1114,6 @@
 	};
 };
 
-&sound_card {
-	dais = <&rsnd_port0	/* ak4613 */
-		&rsnd_port1	/* HDMI0  */
-		&rsnd_port2	/* pcm3168a playback */
-		&rsnd_port3	/* pcm3168a capture  */
-		&rsnd_port4	/* si468x */
-		&rsnd_port5	/* wl18xx */
-		>;
-};
-
 &ssi4 {
 	shared-pin;
 };
-- 
2.7.4

